### DISCUSSION TITLE
issue generate install katana on windows 10

### DISCUSSION THREAD

**Discussion by d4715** [projectdiscovery/katana#623] [Q&A]
i used this command go install github.com/projectdiscovery/katana/cmd/katana@latest for install katana in windows and i got this error so please help me (please check screen shot which is attached)
![1](https://github.com/projectdiscovery/katana/assets/11803332/7ecc8293-45c9-4416-b74b-7694964e4b68)

github.com/smacker/go-tree-sitter/javascript: build constraints exclude all Go files in C:\Users\$$HumaL$$\go\pkg\mod\github.com\smacker\go-tree-sitter@v0.0.0-20220628134258-ac06e95cfa11\javascript
# github.com/smacker/go-tree-sitter
C:\Users\$$HumaL$$\go\pkg\mod\github.com\smacker\go-tree-sitter@v0.0.0-20220628134258-ac06e95cfa11\iter.go:17:18: undefined: Node
C:\Users\$$HumaL$$\go\pkg\mod\github.com\smacker\go-tree-sitter@v0.0.0-20220628134258-ac06e95cfa11\iter.go:21:21: undefined: Node
C:\Users\$$HumaL$$\go\pkg\mod\github.com\smacker\go-tree-sitter@v0.0.0-20220628134258-ac06e95cfa11\iter.go:25:20: undefined: Node
C:\Users\$$HumaL$$\go\pkg\mod\github.com\smacker\go-tree-sitter@v0.0.0-20220628134258-ac06e95cfa11\iter.go:30:26: undefined: Node
C:\Users\$$HumaL$$\go\pkg\mod\github.com\smacker\go-tree-sitter@v0.0.0-20220628134258-ac06e95cfa11\iter.go:34:20: undefined: Node
C:\Users\$$HumaL$$\go\pkg\mod\github.com\smacker\go-tree-sitter@v0.0.0-20220628134258-ac06e95cfa11\iter.go:38:32: undefined: Node
C:\Users\$$HumaL$$\go\pkg\mod\github.com\smacker\go-tree-sitter@v0.0.0-20220628134258-ac06e95cfa11\iter.go:43:9: undefined: Node
C:\Users\$$HumaL$$\go\pkg\mod\github.com\smacker\go-tree-sitter@v0.0.0-20220628134258-ac06e95cfa11\iter.go:46:18: undefined: Node
C:\Users\$$HumaL$$\go\pkg\mod\github.com\smacker\go-tree-sitter@v0.0.0-20220628134258-ac06e95cfa11\iter.go:68:40: undefined: Node

**Comment by MetzinAround** [comment]
Looks like you're getting help over here: https://github.com/projectdiscovery/katana/issues/624. IO'll close this one in favor of the issue linked.
